change/@ni-eslint-config-angular-9ad7619f-6545-4755-83fa-124156f09929.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "major",
+  "comment": "Update to Angular 17",
+  "packageName": "@ni/eslint-config-angular",
+  "email": "7282195+m-akinc@users.noreply.github.com",
+  "dependentChangeType": "patch"
```

Comment:
Marking approve but keeping this comment open to avoid merging until release train starts

Reply:
Release train departing the station. 🚊